### PR TITLE
UAV_tech_5in_Freestyle_650-750 -- TWEAKS

### PR DESCRIPTION
**Pull request by spatzengr**
Updated to Community and tweaked Dyn Idle so it sets correctly without having to set RPM filter to Enable.  Also upped anti-gravity.

### CODE REVIEW

File: index.json [hunk 1]
```diff
@@ -145,49 +146,72 @@
...
       "Justice - Gary Kent",
       "Jye Smith",
-      "Mark Spatz (UAV Tech)",
+      "KMfpv",
+      "LeadFingers",
       "MinChan Kim",
+      "mouseFPV",
+      "MrAlucardDante",
       "NemesisXB",
       "Sjoer van der Ploeg",
```
Comment (haslinghuis) [inline index.json:180]: Sjoerd ???

File: index_hash.txt
Comment (sugaarK) [inline index_hash.txt:1]: Don’t update the Infex file It gets taken care of automatically

Comment (spatzengr) [reply index_hash.txt:1]: I know, but I need to for checking my changes when referring to this branch in the Configurator.

My understanding from @limonspb is it doesn't matter, it updates automatically again once the PR is merged?  So it is ok to, but not needed for the PR purpose?

Comment (sugaarK) [reply index_hash.txt:1]: > I know, but I need to for checking my changes when referring to this branch in the Configurator.
> 
> My understanding from @limonspb is it doesn't matter, it updates automatically again once the PR is merged? So it is ok to, but not needed for the PR purpose?

You can use this instead and it checks not indexes 


`node indexer/check.js `

Comment (limonspb) [reply index_hash.txt:1]: @spatzengr yeah we sort of making everyone to submit just presets, not the indexes. If everyone submits also indexes that will make merge conflicts eventually either here, or in the forks.

Comment (spatzengr) [reply index_hash.txt:1]: @limonspb @sugaarK done.

**Review by haslinghuis — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Pull request merged**